src/dialogs/config-flow/step-flow-abort.ts

```diff
@@ -26,12 +26,20 @@ class StepFlowAbort extends LitElement {
 
   protected render(): TemplateResult | void {
     return html`
-      <h2>Aborted</h2>
+      <h2>
+        ${this.hass.localize(
+          "ui.panel.config.integrations.config_entry.aborted"
```

Comment:
```suggestion
          "ui.panel.config.integrations.config_flow.aborted"
```

Reply:
Resolved in https://github.com/home-assistant/home-assistant-polymer/pull/4140/commits/3cd31a26a03952acb61c9c8b02548fb0d3fda182